lib/sql.reserved.js

```diff
@@ -0,0 +1,532 @@
+'use strict';
+
+module.exports = [
```

Comment:
I think you should provide [a link to the list](https://www.postgresql.org/docs/current/static/sql-keywords-appendix.html) here.

Reply:
Also, I don't see the first entry from the list here.
UPD: Seems that some of the other entries are missing as well, did you omit them on purpose or is this done by mistake?